teuthology/lock/ops.py

```diff
@@ -204,21 +204,19 @@ def unlock_one(ctx, name, user, description=None):
         while proceed():
             try:
                 response = requests.put(uri, json.dumps(request))
-                break
+                if response.ok:
+                    log.debug('unlocked: %s', name)
```

Comment:
```suggestion
                    log.info('unlocked: %s', name)
```
I don't think this is the right time to change how unlocking is logged

Reply:
Agreed. I'll fix this.